packages/server-wallet/deployment/deploy.ts

```diff
@@ -15,9 +15,10 @@ export type TestNetworkContext = {
 
 export async function deploy(): Promise<TestNetworkContext> {
   const ethereumPrivateKey = ETHERLIME_ACCOUNTS[0].privateKey;
+  const ganachePort = Number(process.env.GANACHE_PORT);
```

Comment:
These changes are part of a preceding [PR](https://github.com/statechannels/statechannels/pull/3422)